concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java

```diff
@@ -132,6 +134,21 @@ public Iterator<AccountAddress> getAccountList(final BlockHashInput input) {
         return to(grpcOutput, ClientV2MapperExtensions::to);
     }
 
+    /**
+     * Retrieves the next available {@link AccountNonce} for an account.
+     * This is the {@link AccountNonce} to use for a future transaction
+     * E.g. when using {@link Client#sendTransaction(Transaction)}
+     *
+     * @param address The {@link AccountAddress} of the account to retrieve the next available account nonce for.
+     * @return The next {@link AccountNonce}
+     */
```

Comment:
```suggestion
     *
     * Note that when this function is queried with an non existent account, then it will report the next available 
     * account nonce to be 1 and all transactions as finalized.
     */
```